heron/tools/ui/src/python/BUILD

```diff
@@ -13,6 +13,7 @@ pex_library(
         "jinja2==2.11.2",
         "aiofiles==0.5.0",
         "uvicorn==0.11.7",
+        "uvloop==0.14.0",
```

Comment:
is this the only BUILD file with uvicorn in it? If so, 👍

Reply:
Yup. Looks like only in Heron UI.